mantidimaging/gui/windows/main/view.py

```diff
@@ -109,6 +110,7 @@ class MainWindowView(BaseMainWindowView):
     nexus_save_dialog: NexusSaveDialog | None = None
     add_to_dataset_dialog: AddImagesToDatasetDialog | None = None
     move_stack_dialog: MoveStackDialog | None = None
+    stack_properties_dialog: StackPropertiesDialog | None = None
```

Comment:
This is no longer needed, I think its left over from when there was a `self.stack_properties_dialog`